cloud/cloud.txt

```diff
@@ -0,0 +1 @@
+Simply put, cloud computing is the delivery of computing services—including servers, storage, databases, networking, software, analytics, and intelligence—over the Internet (“the cloud”) to offer faster innovation, flexible resources, and economies of scale. You typically pay only for cloud services you use, helping lower your operating costs, run your infrastructure more efficiently and scale as your business needs change.
```

Comment:
Only markdown files `.md`  are allowed, so change `.txt` to `.md`
Also, use following format: 

```
# Question? #flashcard #cloud
Answer 
--- 
```